ush/ufsda/stage.py

```diff
@@ -303,6 +306,30 @@ def gdas_single_cycle(config):
             target_file = target_file.replace('nc4', 'txt')
             r2d2_config['target_file_fmt'] = target_file
             ufsda.r2d2.fetch(r2d2_config)
+    # if hybvar, link ensemble members to DATA
+    if config['DOHYBVAR']:
+        for imem in range(1, config['NMEM_ENKF']+1):
+            memchar = f"mem{imem:03d}"
+            logging.info("Stage hybvar ens {memchar}")
+            rst_dir = os.path.join(config['COMIN_GES_ENS'], memchar, 'atmos', 'RESTART')
+            ges_dir = os.path.join(config['COMIN_GES_ENS'], memchar, 'atmos', 'RESTART_GES')
+            jedi_bkg_mem = os.path.join(config['DATA'], 'ens')
+            jedi_bkg_dir = os.path.join(config['DATA'], 'ens', memchar)
+
+            mkdir(jedi_bkg_mem)
+
+            # copy RESTART to RESTART_GES
+            if not os.path.exists(ges_dir):
```

Comment:
Last comment before approval (sorry!)

So this section checks to see if the path doesn't exist, and then the try/except also does the `FileExistsError`. Is this just to be extra cautious in case it is created in this split second?
Or is this not doing what you want it to do? What about if the directory exists but it is not 'correct'? In this case it would not even get to the try/except and copy section.

Reply:
The only thing that I can think of is if the analysis (RESTART) files are updated directly rather than passing increments. So then you keep the 'old' RESTART files as the guess and the new ones get updated and then used for the next forecast. We do this in the aerosol DA work right now because the model can't read the aerosol increments, but we can use the existing infrastructure for Gaussian increments after regridding for the ATM app, so no, I don't really see a point in having a `RESTART_GES` either.

Reply:
Using `background_ens` for EnVar and LETKF proved to be more involved than I anticipated.

`/parm/atm/lgetkf/lgetkf.yaml` includes `/RESTART` in the local path to each ensemble member.   `/parm/atm/berror/hybvar_gsibec.yaml` and  `parm/atm/berror/hybvar_bump.yaml` do not include `/RESTART` in the path.  I opted to remove `/RESTART` from `lgetkf.yaml`.   

There's another difference in the local path to the ensemble backgrounds for EnVar and LETKF.   The EnVar path use `ens`.   The LETKF path uses  `bkg`.   Logic was added to `background_ens` in `stage.py` to handle this.  

These changes were committed at 3904f75.

I currently have
```
    dohybvar = config['DOHYBVAR']
    bkgdir = 'ens'
    if not dohybvar: bkgdir = 'bkg'
```
in `background_ens`.   Is there a clever way to compress this to fewer lines?   My google searches have not yet turned up anything.

Reply:
No I think that's about as compressed as it can get, unless you only use `dohybvar` once, then you can replace it with `config['DOHYBVAR']` on the last line and remove the first?

Reply:
Good suggestion.  Decide to directly use `config['DOHYBVAR']`.  Also clean up comments.  These changes committed at 854ed4b.

Manually reran 3dvar, 3denvar, letkf, 3dhofx, 4dhofx via `run_jedi_exe.py` on Orion.  All successfully run to completion.  

Manually ran 49 ctests within g-w `develop`.   All tests pass except one.  `test_gdasapp_atm_jjob_ens_run` failed.   Need to update g-w `scripts/exgdas_global_atmos_ensanal_run.sh` to account for removal of `RESTART` in ensemble member background path.   This g-w change will be addressed in g-w PR [#1373](https://github.com/NOAA-EMC/global-workflow/pull/1373).

Reply:
Ok so since we have a chicken/egg problem and a queue of PRs in g-w, do we want to wait for that or merge this one and accept a failing test for the short term?

Reply:
Ideally we update g-w `develop` at the same time we merge this PR into GDASApp `develop`.   This won't happen.  

My preference is to see this PR enter GDASApp `develop` sooner rather than later.  I'm not keen on maintaining `feature/3denvar` until g-w is ready.  I've been maintaining several g-w branches for a few weeks waiting for PRs [#1370](https://github.com/NOAA-EMC/global-workflow/pull/1370), [#1372](https://github.com/NOAA-EMC/global-workflow/pull/1372), and [#1373](https://github.com/NOAA-EMC/global-workflow/pull/1373) to be reviewed and merged into g-w `develop`.

Reply:
Works for me, will do another review right now